kuma/javascript/src/newsletter.jsx

```diff
@@ -207,44 +219,32 @@ export default function Newsletter() {
                     <fieldset className="newsletter-fields">
                         <input
                             type="hidden"
-                            id="fmt"
                             name="fmt"
                             value={newsletterFormat}
                         />
                         <input
                             type="hidden"
-                            id="newsletterNewslettersInput"
                             name="newsletters"
                             value={newsletterType}
                         />
-                        <div
-                            ref={errorsRef}
-                            id="newsletter-errors"
-                            className={
-                                errorsArray ? 'newsletter-errors' : 'hidden'
-                            }
-                            aria-hidden={errorsArray ? true : false}
-                        >
-                            <ul className="errorlist">
-                                {errorsArray &&
-                                    errorsArray.map((error, index) => (
+                        {errors.length > 0 && (
+                            <div className="newsletter-errors">
+                                <ul className="errorlist">
+                                    {errors.map((error, index) => (
                                         <li key={index}>{error}</li>
```

Comment:
I don't know where my comment went but this is wrong. It's not correct to use array indices as the key. It can trigger that the node isn't updated in the virtual DOM if the *content* changes.

Reply:
It's up here (as well as my response):
https://github.com/mozilla/kuma/pull/5772#discussion_r322723286